waddrmgr/common_test.go

```diff
@@ -53,6 +53,149 @@ var (
 
 	// waddrmgrNamespaceKey is the namespace key for the waddrmgr package.
 	waddrmgrNamespaceKey = []byte("waddrmgrNamespace")
+
+	// 	expectedExternalAddrs is the list of expected external addresses
+	// 	generated from the seed
+	expectedExternalAddrs = []expectedAddr{
```

Comment:
As a suggestion on how to organize this data, I would combine all of the relevant tables into a single slice, and make subslices for groups of data.  So something like:

```
var expectedAddrs []expectedAddr{
    // ...
}

var expectedExternalAddrs = expectedAddrs[0:5]

var expectedInternalAddrs = expectedAddrs[5:10]
```
And then also remove the string slices of expected addrs, since those are not necessary (the address strings are already in the `expectedAddr` structs.